pkg/virtctl/vm/vm.go

```diff
@@ -125,5 +125,6 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Error: VirtualMachineInstance '%s' is already %s", vmiName, stateMsg)
 	}
 
+	cmd.Printf("The request to %s VM was sent successfully\n", o.command)
```

Comment:
just a question regarding wording, would the following be better:
```%vmiName has been scheduled to %command``` ?

Reply:
I did have that thought too.